src/components/App/App.js

```diff
@@ -19,6 +20,8 @@ const App = () => {
           <Route path='/database' component={DatabaseMenu} />
           <Route path='/profiles' component={Profiles} />
           <Route path='/passport' component={Passport} />
+          {/* below is test-only for specific incident view */}
+          <Route path='/test' component={ChartView} />
         </Switch>
```

Comment:
I'm guessing we'll need to pass an id into this component to be able to get the data from a network request?